src/meghaJJune20/Number.java

```diff
@@ -0,0 +1,57 @@
+package meghaJJune20;
+
+public class Number {
+			
+	double sum(double x,double y,double z)
+	{
+		double totalsum=x+y+z;
+		System.out.println("Sum Total is : " +totalsum);
+		return totalsum;
+		
+	}
+	double average(double x,double y,double z)
+	{
+		double avg=(x+y+z)/3;
```

Comment:
you should say averageAnswer = sum(x,y,z)/3; Otherwise there is no need of sum method. we asked you call sum method from average. Please correct it.

Reply:
Updated as per comments